### DISCUSSION TITLE
Rotating objects with adhesive or lubricating contact in between

### DISCUSSION THREAD

**Discussion by malih1186** [idaholab/moose#27717] [Q&A Modules: Solid mechanics]
### Check these boxes if you have followed the posting rules.

- [X] Q&A General is the most appropriate section for my question
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (screenshots as a last resort, triple back quotes around pasted text)

### Question

Hello, 
I have a fluid mechanics background and I am new to MOOSE, I have seen the tutorials and searched the application website but I am still unsure if the problem I have can be simulated with MOOSE. 
I have a stationary and a rotating object (rotating at high RPM) with a very small gap in between. I have searched for FE analysis in Moose but I am unsure about the contact and how to set it up when one object is rotating (to prevent clash) and the mesh is deforming.

I appreciate if you can let me know in case this type of simulation is possible in MOOSE and give me some directions. 
/Mali

**Comment by lindsayad** [comment]
I am not aware of any work in this area. @bwspenc @dschwen @jiangwen84 @recuero are you guys?
